prep-html-links/contact-us/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <div><a href="contact-us.html">Click here to go to the contact page</a>
+</body>
```

Comment:
Hi Jay, unfortunately, if you click this link it does not take you to the contact us page. Try removing all of your extra files and then try to get the `index.html` to link to the `contact-us/contact-us.html` page as well as the `contact-us/contact-us.html` to link back to the `index.html`.